daemon/algod/api/algod.oas2.json

```diff
@@ -1146,6 +1147,12 @@
               "$ref": "#/definitions/ErrorResponse"
             }
           },
+          "404": {
```

Comment:
Is this 404 case being added for future compatibility? I don't see this is as current possibility.

Reply:
Thanks, removed.